src/util/DeleteComponent.tsx

```diff
@@ -50,7 +50,7 @@ function DeleteComponent({
 
     const renderCannotDeleteDialogModal = () => {
         return (
-            <ConfirmationDialog className="confirmation-dialog__body--w-360">
+            <ConfirmationDialog className="confirmation-dialog__body--w-400">
```

Comment:
is this change intentional?

Reply:
Yes. Made the change as part of the UX suggestions.